OrbitCore/SymbolHelper.cpp

```diff
@@ -0,0 +1,166 @@
+#include "SymbolHelper.h"
+
+#include <fstream>
+
+#include "ElfFile.h"
+#include "OrbitBase/Logging.h"
+#include "Path.h"
+#include "Pdb.h"
+
+std::vector<std::string> SymbolHelper::collector_symbol_locations = {
+    "/home/cloudcast/",  "/home/cloudcast/debug_symbols/",
+    "/mnt/developer/",   "/mnt/developer/debug_symbols/",
+    "/srv/game/assets/", "/srv/game/assets/debug_symbols/"};
```

Comment:
This should probably not be exposed on github without prior clearance.

Reply:
See Alens message in Chat

Reply:
there are a lot of messages in the chat :)

Reply:
Alen said it's fine! :-D